app/views/shared/menu/_cart_sidebar.html.haml

```diff
@@ -16,9 +16,9 @@
           %td.image
             %img{'ng-src' => '{{ line_item.variant.thumb_url }}'}
           %td
-            %span {{ line_item.variant.extended_name }}
+            %span {{ line_item.variant.extended_name | truncate: max_characters }}
```

Comment:
It'd be better in terms of UX if we used ellipsis too: https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

Reply:
The `truncate` filter is a custom OFN one, and it already inserts an ellipsis.

Reply:
awesome!